stdlib/experimental/prometheus/prometheus.flux

```diff
@@ -1,7 +1,17 @@
 package prometheus
+import "universe" 
 
 // scrape enables scraping of a prometheus metrics endpoint and converts 
 // that input into flux tables. Each metric is put into an individual flux 
 // table, including each histogram and summary value.  
 builtin scrape
 
+// histogramQuantile enables the user to calculate quantiles on a set of given values
+// This function assumes that the given histogram data is being scraped or read from a 
+// Prometheus source. 
+histogramQuantile = (tables=<-, field, quantile) => 
+    tables
+        |> filter(fn: (r) => r._measurement == "prometheus" and r._field == field)
+        |> group(mode: "except", columns: ["le", "_value"]) 
```

Comment:
Can you remind me how the data comes grouped from the `scrape` call? Is `le` in the group key and that is what we are trying to fix here? If so maybe we should use the experimental.group function and subtract the `le` column from the group key but otherwise not change the group key. This will enable users to compute historgram quantiles across many different fields based on what ever grouping they setup before call this function.

Reply:
I'm not sure that `experimental.group` has the capability to subtract a column from the `groupKey` and leave all others as is. I just spoke with @wolffcm and he said there's not a mode for that yet. 

I'll open an issue to add another mode that would make it easy to remove one column from the `groupKey`, but for now I think we might have to stick with regular `group()`.